pkg/providers/aws/credentials_test.go

```diff
@@ -73,8 +80,11 @@ func TestCredentialMinterManager_ReconcileProviderCredentials(t *testing.T) {
 	}
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			cm := NewCredentialManager(tc.client).(*CredentialMinterCredentialManager)
-			awsCreds, err := cm.ReconcileProviderCredentials(context.TODO(), "testNamespace")
+			cm, err := NewCredentialManager(tc.client)
+			if err != nil {
```

Comment:
The results in fatal here but not in a previous test. Any reason for the difference?